requirements.txt

```diff
@@ -6,3 +6,5 @@ mock
 falcon==1.1.0
 gunicorn==19.6.0
 mysqlclient==1.3.10
+hypothesis==3.7.3
+pytz
```

Comment:
Hypothesis needed it for something I think... Can build without it now and can't remember why so will remove.